src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
data generated equals, hashcode and toString including ALL fields. What about Category? Do you need this field as a part of these methods?